.github/workflows/schedule.yml

```diff
@@ -153,7 +153,7 @@ jobs:
 
       - name: OPA tests
         run: |
-          curl -L -o opa https://openpolicyagent.org/downloads/v0.45.0/opa_linux_amd64_static
+          curl -L -o opa https://openpolicyagent.org/downloads/v0.63.0/opa_linux_amd64_static
```

Comment:
Could you apply the change from #7273 to this workflow, so that we don't need to hardcode the version anymore? (I evidently missed this workflow when I made that PR.)

Reply:
Changed